guides/hiring/remote_test/csharp_marking_guide.md

```diff
@@ -0,0 +1,28 @@
+# Marking guide
+
+60 marks in total. There are no half marks. 80% (48 marks) required to pass.
+
+ - [2] The timeline appeared on the homepage
+ - [2] The form appeared on the homepage
+ - [3] The form displayed specific error if the message is empty
+ - [3] The form displayed specific error if the message is more than 150 characters
+ - [3] The timeline displayed message if form submitted without errors
+ - [3] The page displayed a message if form submitted without errors
+ - [2] The timeline appeared below the form
+ - [3] The timeline is ordered in descending chronological order
+ - [2] Each item in the timeline contains a message
+ - [2] Each item in the timeline contains the time of submission
+ - [2] Each item in the timeline contains a way to delete the post
+ - [3] Message can contain newlines, new lines are rendered
+ - [3] Time is displayed in format of "20 minutes ago", "5 days ago"
+ - [3] Clicking delete button results in item being deleted
+ - [3] Clicking delete button results in message being displayed
+ - [3] Git was used to version control
```

Comment:
If we make this the language agnostic marking guide, perhaps we should drop specifying "Git"... I personally wouldn't mind if someone pushed up a Mercurial repo, for example